### ISSUE TITLE
Not able to run LLVM10 clangd and clang-tidy

### ISSUE THREAD

**Issue by julyip** [clearlinux/distribution#1932]
Hi,
It seems like LLVM10 clangd and clang-tidy is not linked properly. As a result, it gives "CommandLine Error" when you run it. For example:
```
$clang-tidy -v
: CommandLine Error: Option 'filter' registered more than once!
LLVM ERROR: inconsistency in registered CommandLine options
$clangd -v
: CommandLine Error: Option 'filter' registered more than once!
LLVM ERROR: inconsistency in registered CommandLine options
```
If you have any questions/comments, please feel free to let me know.
Thanks,
Julian

**Comment by thiagomacieira** [comment]
Must be one of our patches changing the command-line options that I didn't rebase properly.

**Comment by thiagomacieira** [comment]
Not a patch. Looks like I caused the Clang tools (but apparently not Clang itself) to link to libclang-cpp.so and the individual libs, which causes almost all options to be duplicated.

**Comment by thiagomacieira** [comment]
Fixed in clearlinux-pkgs/llvm#10.0.0-120. Should be in Clear 33020.

**Comment by julyip** [comment]
> Fixed in clearlinux-pkgs/llvm#10.0.0-120. Should be in Clear 33020.

Thank you
